python_modules/libraries/dagster-dbt/dagster_dbt/components/dbt_project/scaffolder.py

```diff
@@ -13,38 +13,50 @@
 class DbtScaffoldParams(BaseModel):
```

Comment:
Aside, I was thinking about this yesterday, but part of me finds it odd that we have to specify scaffold params independently from the attributes on the Component itself, since these are also being provided there.

Reply:
It's a bit odd for sure but it allows us to only expose a subset of the component configuration that makes sense (also stuff like `init` is not found on the component as it's something you do before the project even exists)